lib/js/components/Tile/Tile.vue

```diff
@@ -35,18 +44,132 @@
 @import '../../../styles/settings/radiuses';
 @import '../../../styles/settings/colors/tokens';
 
+$tile-colors: (
+	'neutral': (
+		'default': (
+			'background': $color-neutral-background,
+			'background-hover': $color-neutral-background-hovered,
+			'eyebrow-text': $color-neutral-text-weak,
+			'icon': $color-neutral-icon,
+			'icon-interactive': $color-primary-icon,
+		),
+		'disabled': (
+			'background': $color-neutral-background-disabled,
+			'eyebrow-text': $color-neutral-text-weak-disabled,
+			'icon': $color-neutral-icon-disabled,
+			'icon-interactive': $color-primary-icon-disabled,
+		),
+	),
+	'primary': (
+		'default': (
+			'background': $color-primary-background,
+			'background-hover': $color-primary-background-hovered,
+			'eyebrow-text': $color-primary-text,
+			'icon': $color-primary-icon,
+			'icon-interactive': $color-primary-icon,
+		),
+		'disabled': (
+			'background': $color-primary-background-disabled,
+			'eyebrow-text': $color-primary-text-disabled,
+			'icon': $color-primary-icon-disabled,
+			'icon-interactive': $color-primary-icon-disabled,
+		),
+	),
+	'success': (
+		'default': (
+			'background': $color-success-background,
+			'background-hover': $color-success-background-hovered,
+			'eyebrow-text': $color-success-text,
+			'icon': $color-success-icon,
+			'icon-interactive': $color-success-icon,
+		),
+		'disabled': (
+			'background': $color-success-background-disabled,
+			'eyebrow-text': $color-success-text-disabled,
+			'icon': $color-success-icon-disabled,
+			'icon-interactive': $color-success-icon-disabled,
+		),
+	),
+	'fail': (
+		'default': (
+			'background': $color-fail-background,
+			'background-hover': $color-fail-background-hovered,
+			'eyebrow-text': $color-fail-text,
+			'icon': $color-fail-icon,
+			'icon-interactive': $color-fail-icon,
+		),
+		'disabled': (
+			'background': $color-fail-background-disabled,
+			// TODO IT-4795 change to $color-fail-text-disabled
+			'eyebrow-text': var(--raw-orange-200),
+			'icon': $color-fail-icon-disabled,
+			'icon-interactive': $color-fail-icon-disabled,
+		),
+	),
+);
+
+@mixin setColors($root, $color-map) {
+	background-color: map-get($color-map, 'background');
+
+	#{$root}__eyebrowText {
+		color: map-get($color-map, 'eyebrow-text');
+	}
+
+	#{$root}__iconLeft,
+	#{$root}__iconRight {
+		color: map-get($color-map, 'icon');
+	}
+
+	&.-interactive {
+		#{$root}__iconRight {
+			color: map-get($color-map, 'icon-interactive');
```

Comment:
wrong color for `neutral primary disabled`. Should be `$color-primary-icon-disabled`, on storybook I see `$color-neutral-icon-disabled`

Reply:
fixed in https://github.com/bethinkpl/design-system/pull/185/commits/4159900dd26c3a4bdf020905c6e9f0e9909b37b6